deploy/kubernetes/base/controller.yaml

```diff
@@ -21,6 +21,30 @@ spec:
       tolerations:
         - key: CriticalAddonsOnly
           operator: Exists
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: "beta.kubernetes.io/os"
+                operator: In
+                values:
+                - linux
```

Comment:
We have the `nodeSelector` above, is that still needed?

Reply:
I'll remove the  affinity for `linux` and let it to the node selector.